backend/app/services/prompt_optimization.py

```
@@ -0,0 +1,302 @@
+    applied = 0
+
+    for change in changes:
+        try:
+            if change.type == "ghost":
+                # Apply to Ghost
+                ghost = (
+                    db.query(Kind)
+                    .filter(
+                        Kind.id == change.id,
+                        Kind.kind == "Ghost",
+                        Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)